Source/OrchBase/OrchBaseLink.cpp

```diff
@@ -144,7 +144,178 @@ if (pT->linked) {                      // Task already linked?
 
 */
 // Place and set the link flag if the placement was successful.
-if (!pPlace->Place(pT)) pT->LinkFlag();
+// if (!pPlace->Place(pT)) pT->LinkFlag();

```

Comment:
There is a further argument for doing this in that as things stood, there was a "link /nser" command which uploaded the NameServer, currently that's being done in link /link, but if we also have a separate command the NameServer could be reloaded if necessary. One question remains though: if this is done, link /nser would have to delete the NameServer information. Should it wipe out everything or just a given task?

Whatever the case is, again maybe this should wait until after the initial pull has been completed, because there is some restructuring involved.